VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-id/copy-id.component.scss

```diff
@@ -10,11 +10,7 @@
     left: 139px;
   }
 .copyIDbox {
-    font-family: "GDS Transport", Arial, sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    font-weight: 400;
-    font-size: 14px;
+    @include govuk-font(14, $weight: 'bold', $tabular: false, $line-height: false);
```

Comment:
should this be bold? I thought this was regular weighting?

Reply:
updated to regular,